frontend/webadmin/modules/webadmin/src/main/java/org/ovirt/engine/ui/webadmin/widget/host/HostNetworkInterfaceListViewItem.java

```diff
@@ -182,8 +183,10 @@ private IsWidget createRxColumn(VdsNetworkInterface hostInterface) {
         divider.addStyleName(RATE_DIVIDER);
         rxCol.add(divider);
         dl = Document.get().createDLElement();
+        BigInteger receivedBytes = hostInterface.getStatistics().getReceivedBytes() != null ?
+                hostInterface.getStatistics().getReceivedBytes() : new BigInteger("0"); //$NON-NLS-1$
         addDetailItem(templates.sub(constants.rxTotal(), constants.bytes()),
-                SafeHtmlUtils.fromString(totalRenderer.render(hostInterface.getStatistics().getReceivedBytes())), dl);
+                SafeHtmlUtils.fromString(totalRenderer.render(receivedBytes)), dl);
```

Comment:
The renderer should be able to handle the `null` and display `[N/A]`. If you want to change it to `0` - wouldn't it be better to send the `0` from the backend query (or even set `0` as the default in the database)?

Reply:
Done with N\A after consulting with QE